testing/trino-benchto-benchmarks/src/main/resources/sql/presto/tpcds/q77.sql

```diff
@@ -88,7 +88,7 @@ SELECT
 FROM
   (
    SELECT
-     '${database}.${schema}.store channel' "channel"
```

Comment:
nit: separate commit

Reply:
This is a separate commit (apart from unintentional whitespace changes - I will revert them).